modules/ROOT/pages/building-https-proxy.adoc

```diff
@@ -0,0 +1,147 @@
+= Building an HTTPS API Proxy
+
+API Manager 2.x lets you use the secrets you stored in your secret group to build an HTTPS based API for your CloudHub or Hybrid deployments. +
+By configuring a TLS context for your inbound and outbound traffic, you can ensure your traffic's security.
+
+The secrets manager integration is only supported by API Proxies running Mule 4, and only for CloudHub, Hybrid, and standalone deployments. +
+If you want to build and HTTPS Proxy in Mule 3, follow the xref:runtime-fabric::building-an-https-service.adoc[Building an HTTPS Service] tutorial. 
+
+== Prerequisites
+
+To configure your HTTPS API Proxy using secrets stored in your secret group, you need to meet the following requirements:
+
+* Secrets Manager Permissions enabled for your user. +
+To configure an existing TLS context for the API Proxy, you need the *Read secrets metadata* permission.
+Review the xref:anypoint-security::asm-permission-concept.adoc[Secrets Manager Permissions] reference for more information.
+* Make sure your user has an *API Versions Owner* role.
```

Comment:
`API Versions Owner` isn't available in post crowd world. I would change it for `Make sure your user has at least *Manage API Configuration*`

Reply:
Updated https://github.com/mulesoft/docs-api-manager/pull/21/commits/604add4830d327f5268bba155e171e8a0ad9a776